src/D8/EckEntityTrait.php

```diff
@@ -0,0 +1,196 @@
+<?php
```

Comment:
the name of the trait is usually a module name.

Can this please be changed to `EckTrait.php`

Reply:
@AlexSkrypnyk done thanks for the review